src/Erl/Data/Map/Gen.purs

```diff
@@ -0,0 +1,24 @@
+module Erl.Data.Map.Gen where
```

Comment:
Was going to try use quickcheck like in `ordered-collections` but I assume we'd need our own implementation because of different representations of Float in Erlnag vs JS?  https://github.com/purescript/purescript-quickcheck/blob/master/src/Test/QuickCheck/Gen.js